### ISSUE TITLE
[EXP]: Logits on test set for ALBEF on Sarc Data Splits

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#73]
### Description

Currently we are missing logits on test set of ALBEF models trained on different splits.

**Comment by lwaekfjlk** [comment]
fixed
